internal/wire/transport_parameters.go

```diff
@@ -478,7 +482,8 @@ func (p *TransportParameters) ValidFor0RTT(saved *TransportParameters) bool {
 		p.InitialMaxData >= saved.InitialMaxData &&
 		p.MaxBidiStreamNum >= saved.MaxBidiStreamNum &&
 		p.MaxUniStreamNum >= saved.MaxUniStreamNum &&
-		p.ActiveConnectionIDLimit == saved.ActiveConnectionIDLimit
+		p.ActiveConnectionIDLimit == saved.ActiveConnectionIDLimit &&
+		p.MaxDatagramFrameSize >= saved.MaxDatagramFrameSize
```

Comment:
This is relying on the fact that `protocol.InvalidByteCount < 0`, but that's not a guarantee.